input/pagecontent/changes.md

```diff
@@ -13,6 +13,9 @@ This change log documents the significant updates and resolutions implemented fr
   - Patient.extension:interpreterRequired [FHIR-45186](https://jira.hl7.org/browse/FHIR-45186)
   - Patient.birthDate.extension:birthTime [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
   - Patient.birthDate.extension:accuracyIndicator [FHIR-45028](https://jira.hl7.org/browse/FHIR-45028)
+- Removed Must Support from the following elements in AU Core Practitioner:
+  - Practitioner.communication [FHIR-45018](https://jira.hl7.org/browse/FHIR-45018)
+  - Practitioner.qualification, Practitioner.qualification.identifier, Practitioner.qualification.code, Practitioner.qualification.period, Practitioner.qualification.issuer: [FHIR-45118](https://jira.hl7.org/browse/FHIR-45118), [FHIR-44587](https://jira.hl7.org/browse/FHIR-44587), [FHIR-44588](https://jira.hl7.org/browse/FHIR-44588)
```

Comment:
Remove mention of FHIR-44588 as it's about the removal of MS from Practitioner.communication

Reply:
Fixed.